flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobResult.java

```diff
@@ -108,22 +118,39 @@ public long getNetRuntime() {
 	 *
 	 * @param classLoader to use for deserialization
 	 * @return JobExecutionResult
-	 * @throws WrappedJobException if the JobResult contains a serialized exception
+	 * @throws JobExecutionException if the job execution did not succeed
```

Comment:
this method might throws `JobCancellationException` if the job is cancelled, should be documented.